ui/main/src/app/services/i18n.service.ts

```diff
@@ -52,8 +52,11 @@ export class I18nService {
 
     public loadLocale(locale: string): Observable<any> {
         return this.httpClient.get(`${I18nService.localUrl}${locale}.json`).pipe( tap({
-            next: translation => this.translate.setTranslation(locale, translation, true),
-            error: error => console.log(new Date().toISOString(),`Error : impossible to load locale ${I18nService.localUrl}${locale}.json`)
+            next: translation => {
+                this.translate.setTranslation(locale, translation, true);
+                console.log(new Date().toISOString(), 'opfab translation loaded for locale:', locale);
```

Comment:
Should we start using the new OpfabLoggerService instead of console log?

Reply:
You're right. Done.